$MirOS: ports/devel/cvs/patches/patch-src_cvsrc_c,v 1.1 2010/09/15 20:57:00 tg Exp $
--- src/cvsrc.c.orig	Wed Mar 16 15:51:52 2005
+++ src/cvsrc.c	Wed Sep 15 23:39:58 2010
@@ -17,6 +17,8 @@
 #include "cvs.h"
 #include "getline.h"
 
+__RCSID("$MirOS: ports/devel/cvs/patches/patch-src_cvsrc_c,v 1.1 2010/09/15 20:57:00 tg Exp $");
+
 /* this file is to be found in the user's home directory */
 
 #ifndef	CVSRC_FILENAME
@@ -41,6 +43,7 @@ read_cvsrc (int *argc, char ***argv, con
     size_t line_chars_allocated;
 
     char *optstart;
+    int white_len = 0;
 
     int command_len;
     int found = 0;
@@ -96,9 +99,12 @@ read_cvsrc (int *argc, char ***argv, con
 	if (line[0] == '#')
 	    continue;
 
+	while (isspace(line[white_len]))
+		++white_len;
+
 	/* stop if we match the current command */
-	if (!strncmp (line, cmdname, command_len)
-	    && isspace ((unsigned char) *(line + command_len)))
+	if (!strncmp (line + white_len, cmdname, command_len)
+	    && isspace ((unsigned char) *(line + white_len + command_len)))
 	{
 	    found = 1;
 	    break;
@@ -120,7 +126,7 @@ read_cvsrc (int *argc, char ***argv, con
     if (found)
     {
 	/* skip over command in the options line */
-	for (optstart = strtok (line + command_len, "\t \n");
+	for (optstart = strtok (line + white_len + command_len, "\t \n");
 	     optstart;
 	     optstart = strtok (NULL, "\t \n"))
 	{
